### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheTechRobo**
This is a better PR to #201, #201 was getting cluttered (thanks, myself). And, I also just wanted to start from scratch. 

This PR helps make the README.md better in the following areas:

- Fixed broken links
- Fixed grammar
- Made it more easy to read

~TheTechRobo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 # PirateBoxScripts with Modifications for running in a Webserver     
 
-[![Join the chat at https://gitter.im/PirateBox-Dev/PirateBoxScripts_Webserver](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PirateBox-Dev/PirateBoxScripts_Webserver?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-&copy; 2012-2014 [Matthias Strubel](mailto:matthias.strubel@aod-rpg.de) licenced under GPL-3
+[![Join the chat at https://gitter.im/PirateBox-Dev/PirateBoxScripts_Webserver](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PirateBox-Dev/PirateBoxScripts_Webserver?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)  
+
+&copy; 2012-2014 [Matthias Strubel](mailto:matthias.strubel@aod-rpg.de)
```
Comment (MaStr) [inline README.md]: Is it possible, that you change the Copyright  to _"2012-2019"_

Comment (TheTechRobo) [reply README.md]: Will do.

Comment (TheTechRobo) [reply README.md]: Done

**Comment by MaStr** [comment]
thx

**Pull request merged**